### DISCUSSION TITLE
SVG Version of MeshLab Logo

### DISCUSSION THREAD

**Discussion by joshi1983** [cnr-isti-vclab/meshlab#1371] [Show and tell]
I didn't make this in MeshLab but this could be useful for high resolution printing of your MeshLab logo to t-shirts or on the website.  I made it because the only version I could find of the MeshLab logo was in PNG format and the green eye looked very attractive.

![MeshLab-logo](https://user-images.githubusercontent.com/11838191/227018043-d0fa9366-6f96-4b4e-858f-e9c478ae5963.svg)

If you like this and want to use it, I might make a 3D model that could be a nicely branded 3D model to come with the MeshLab installation.

**Comment by cignoni** [comment]
Nice! 
I drew it many years ago (18y!), actually the original svg is in the github repo here:

https://github.com/cnr-isti-vclab/meshlab/blob/main/src/meshlab/images/eye.svg

cheers
p.
